.github/actions/npm-publish/action.yml

```diff
@@ -51,7 +51,7 @@ runs:
           TAG="latest"
         fi
         npm config set "//registry.npmjs.org/:_authToken" "${NPM_TOKEN}"
-        pnpm publish --tag $TAG
+        pnpm publish
```

Comment:
Not adding the latest tag as this is not the latest. Assumably this works, if not, we need to fix the dist-tags post-release.